### PR TITLE
Dump convention tests results to files

### PR DESCRIPTION
**Pull request by kokosing**
Remove expected-result-generator

### CODE REVIEW

**Comment by kokosing** [comment]
CC: @brian-rickman

**Comment by brian-rickman** [comment]
Code looks good.  Just one question: what does it do when a single file contains multiple SQL queries?

**Comment by brian-rickman** [comment]
LGTM

File: tempto-core/src/main/java/com/teradata/tempto/internal/convention/ConventionTestsUtils.java [hunk 1]
```diff
@@ -148,8 +148,12 @@ private static void processPathFromUri(URI uri, Consumer<Path> action)
         }
     }
 
-    public static boolean isDisplayConventionTestResultsRequested() {
-        return System.getProperty(DISPLAY_CONVENTION_TESTS_RESULTS_KEY, FALSE.toString()).equals(TRUE.toString());
+    public static Optional<Path> getConventionTestResultsDumpPath() {
+        String path = System.getProperty(CONVENTION_TESTS_RESULTS_DUMP_PATH_KEY, "");
```
Comment (losipiuk) [inline tempto-core/src/main/java/com/teradata/tempto/internal/convention/ConventionTestsUtils.java]: Change this to
```java
return Optional.ofNullable(System.getProperty(C_T_R_D_P_K))
```

**Comment by losipiuk** [comment]
One minor suggestion. Looks good otherwise.

**Pull request merged**